modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_settings/dl_file_entry_configuration.jsp

```diff
@@ -0,0 +1,108 @@
+				<liferay-ui:message key="dl-file-entry-configuration-name" />
+			</h2>
+		</clay:sheet-header>
+
+		<clay:alert
+			dismissible="<%= true %>"
+			displayType="info"
+			message="changes-will-only-apply-to-new-documents-uploaded"
+		/>
+
+		<clay:sheet-section>
+			<h3 class="c-mb-2 sheet-subtitle text-secondary"><liferay-ui:message key="dl-file-entry-configuration-maximum-file-size-title" /></h3>
+
+			<p class="c-mb-4 text-3 text-secondary">
+				<liferay-ui:message key="dl-file-entry-configuration-maximum-file-size-help" />
+			</p>
+
+			<div>
+				<span aria-hidden="true" class="loading-animation"></span>
+
+				<react:component
+					module="document_library_settings/js/LimitedNumberInput"
+					props='<%=
+						HashMapBuilder.<String, Object>put(
+							"errorMessage", dlFileEntryConfigurationDisplayContext.getPreviewableProcessorMaxSizeLimitExceededErrorMessage()
+						).put(
+							"label", LanguageUtil.get(resourceBundle, "maximum-file-size")
+						).put(
+							"limitValue", dlFileEntryConfigurationDisplayContext.getPreviewableProcessorMaxSizeLimit()
+						).put(
+							"minimumValue", dlFileEntryConfigurationDisplayContext.getUnlimitedPreviewableProcessorMaxSize()
+						).put(
+							"name", liferayPortletResponse.getNamespace() + "previewableProcessorMaxSize"
+						).put(
+							"unlimitedValue", dlFileEntryConfigurationDisplayContext.getUnlimitedPreviewableProcessorMaxSize()
+						).put(
+							"value", dlFileEntryConfigurationDisplayContext.getPreviewableProcessorMaxSize()
+						).build()
+					%>'
+				/>
+			</div>
+		</clay:sheet-section>
+
+		<clay:sheet-section>
+			<h3 class="c-mb-2 sheet-subtitle text-secondary"><liferay-ui:message key="dl-file-entry-configuration-maximum-number-of-pages-title" /></h3>
+
+			<p class="c-mb-4 text-3 text-secondary">
+				<liferay-ui:message key="dl-file-entry-configuration-maximum-number-of-pages-help" />
+			</p>
+
+			<div>
+				<span aria-hidden="true" class="loading-animation"></span>
+
+				<react:component
+					module="document_library_settings/js/LimitedNumberInput"
+					props='<%=
+						HashMapBuilder.<String, Object>put(
+							"errorMessage", dlFileEntryConfigurationDisplayContext.getMaxNumberOfPagesLimitExceededErrorMessage()
+						).put(
+							"label", LanguageUtil.get(resourceBundle, "maximum-number-of-pages")
+						).put(
+							"limitValue", dlFileEntryConfigurationDisplayContext.getMaxNumberOfPagesLimit()
+						).put(
+							"minimumValue", dlFileEntryConfigurationDisplayContext.getUnlimitedMaxNumberOfPages()
+						).put(
+							"name", liferayPortletResponse.getNamespace() + "maxNumberOfPages"
+						).put(
+							"unlimitedValue", dlFileEntryConfigurationDisplayContext.getUnlimitedMaxNumberOfPages()
+						).put(
+							"value", dlFileEntryConfigurationDisplayContext.getMaxNumberOfPages()
+						).build()
+					%>'
+				/>
+			</div>
+		</clay:sheet-section>
+
+		<clay:sheet-footer>
+			<aui:button primary="<%= true %>" type="submit" />
```

Comment:
We can replace this by a clay:button
```
<clay:button
     displayType="primary"
     type="submit"
/>
```

Reply:
Done, see file in  [246eb4d](https://github.com/liferay-lima/liferay-portal/pull/4668/commits/246eb4da6b9acf76cb30db4b8708a22c0eaa9763)

Reply:
Perfect! Thank you